packages/perseus/src/widgets/radio/__stories__/multiple-choice-option.stories.tsx

```diff
@@ -1,6 +1,6 @@
 import * as React from "react";
 
-import Choice from "../multiple-choice-option.new";
+import Choice from "../choice.new";
```

Comment:
Is the [Widgets/RadioNew/Widget Internal Components/Choice](https://650db21c3f5d1b2f13c02952-knpmiiicyz.chromatic.com/?path=/docs/widgets-radionew-widget-internal-components-choice--docs) story supposed to look like this?

<img width="1208" height="723" alt="Screenshot 2025-08-04 at 2 09 12 PM" src="https://github.com/user-attachments/assets/39ffb86f-57b7-46ba-8552-0d0eece40931" />